### PR TITLE
renamed CoBL to IE-CoR; updated home page

### PR DESCRIPTION
**Pull request by Bibiko**

### CODE REVIEW

File: cobl2/templates/dataset/detail_html.mako [hunk 1]
```diff
@@ -8,11 +9,44 @@
...
+
+<h4>The CoR Database System</h4>
+
+<p>CoR is a new database structure for exploring the <strong>Co</strong>gnate <strong>R</strong>elationships between languages in their basic or &#8216;core&#8217; vocabulary.  The core vocabulary covered is a reference list of 170 basic and common word meanings, such as ‘one’, ‘water’, ‘black’, and so on.  CoR uses a new 170 meaning list, based on a combination and adaptation of three lists already widely used in linguistics:  the Swadesh lists (100 and 200), and the Leipzig-Jakarta list (100).  This combination has been heavily revised to optimise it for consistency in both lexeme determination, and cognacy determination within Indo-European.  All 170 meanings have also been precisely (re)defined. </p>
+
+<p>CoR is tailored for qualitative as well as quantitative research purposes, and includes data-exploration websites to search the rich linguistic data covered:  cognate sets, orthography, morphology, phonemic and IPA phonetic transcriptions, and links to further sources.  </p>
```
Comment (xrotwang) [inline cobl2/templates/dataset/detail_html.mako]: I'd leave the explanation of the database system to the demo app of that system, i.e. remove the "The CoR Database System" section here.

Comment (Bibiko) [reply cobl2/templates/dataset/detail_html.mako]: but in this section is explained which meanings are taken and why and how - the last paragraph of this section could be removed maybe - but I think, the section header name could only be changed into `The CoR data` or similar

Comment (xrotwang) [reply cobl2/templates/dataset/detail_html.mako]: Ah, ok. Maybe we just keep the sentence explaining this? The 170 item list isn't fixed in the database system anyway, right? So it shouldn't be described as a defining feature of it?

Comment (Bibiko) [reply cobl2/templates/dataset/detail_html.mako]: ok

**Comment by xrotwang** [comment]
Feel free to merge after editing the sentence about the database system.

**Pull request merged**